cli/test/lib/tasks/__snapshots__/install.spec.ts.snap

```diff
@@ -0,0 +1,282 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`/lib/tasks/install > .start > exits with error when installing on unsupported os > error when installing on unsupported os 1`] = `
+"Error: The Cypress App could not be installed. Your machine does not meet the operating system requirements.
+
+https://on.cypress.io/app/get-started/install-cypress#System-requirements
+
+----------
+
+Platform: win32-ia32
+"
+`;
+
+exports[`/lib/tasks/install > .start > is silent when log level is silent > silent install 1 1`] = `
+"[STARTED] Task without title.
```

Comment:
I left a detailed comment in the spec, but the `Listr` renderer that is used in Vitest is different than the renderer used in Mocha. This is due to TTY not being supported inside of service worker processes, so Vitest defaults back to the VerboseRenderer vs the DefaultRenderer. The VerboseRenderer is likely better for us in a snapshot context because the text printed is deterministic as opposed to icons being constantly replaced in the terminal 

see https://listr2.kilic.dev/renderer/renderer.html#frontmatter-title